### PR TITLE
feat: adding 'try-in-web-ide' GitHub action

### PR DESCRIPTION
**Pull request by ibuziuk**
https://github.com/marketplace/actions/try-in-web-ide

### What does this PR do?
adding 'try-in-web-ide' GitHub action - https://github.com/marketplace/actions/try-in-web-ide 

### What issues does this PR fix or reference?
Related to https://github.com/eclipse/che/issues/20155

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .github/workflows/try-in-web-ide.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Add Web IDE link on PRs
```
Comment (benoitf) [inline .github/workflows/try-in-web-ide.yaml:1]:
```suggestion
#
# Copyright (c) 2021 Red Hat, Inc.
# This program and the accompanying materials are made
# available under the terms of the Eclipse Public License 2.0
# which is available at https://www.eclipse.org/legal/epl-2.0/
#
# SPDX-License-Identifier: EPL-2.0
#
# Add Web IDE link on PRs
```

File: .github/workflows/try-in-web-ide.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  pull_request_target:
+    types: [opened, synchronize]
+
+jobs:
+  add-link:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Web IDE Pull Request Check
+        id: try-in-web-ide
+        uses: redhat-actions/try-in-web-ide@v1
```
Comment (benoitf) [inline .github/workflows/try-in-web-ide.yaml:14]: I'm fine to use @main as well to have always latest new changes

Comment (benoitf) [reply .github/workflows/try-in-web-ide.yaml:14]: FYI for che incubator happy path we publish a next version on every commit
https://github.com/che-incubator/happy-path-tests-action/blob/main/.github/workflows/publish-next.yml

**Review by benoitf — APPROVED** [review]

**Comment by sunix** [comment]
@ibuziuk midnight question: how can we try that ?

**Review by svor — APPROVED** [review]

**Pull request merged**